llvm/include/llvm/ADT/PagedVector.h

```diff
@@ -103,6 +103,12 @@ template <typename T, size_t PageSize = 1024 / sizeof(T)> class PagedVector {
   /// Return the size of the vector.
   [[nodiscard]] size_t size() const { return Size; }
 
+  [[nodiscard]] bool isMaterialized(size_t Index) const {
```

Comment:
Could you add a documentation comment above? This is not a standard function found in most containers so it would be nice to introduce it properly.

Reply:
```suggestion
  /// @return true in case the element at index @a Index belongs to a page which
  /// was already materialised.
  [[nodiscard]] bool isMaterialized(size_t Index) const {
```